docs/infrastructure.md

```diff
@@ -0,0 +1,62 @@
+# Infrastructure
+
+This app is deployed on [GOV.UK PAAS][government-paas].
+
+## Space for the app
+
+You may be given a space by your organisation manager, or need to create one
+for the application.
+
+At the time of writing this app runs on the `govuk_development` organisation
+in the `sandbox` space
+
+## Push the app to the PAAS
+
+As we've not set everything up we don't want to start the app now.
```

Comment:
This reads oddly, like a Trello card for a "create the Pact Broker infrastructure" as if it hasn't been done yet! Maybe rather than delete the whole file (which contains useful context such as which ENV variables are needed) we can delete some of the setup stuff that is no longer needed.

Reply:
I'm not sure this approach works: even the "ENV" part of the file is still specific to creating the app from scratch. Those two variables are also mentioned separately in the README.

Perhaps a sentence at the top would help to explain the guidance is about creating the app from scratch in PaaS. What do you think?

Reply:
OK 👍

Reply:
Amended in https://github.com/alphagov/govuk-pact-broker/pull/123/commits/d88bac4623e2d96ef9917c771725e6e487c69ca7.